src/rules/no-large-snapshots.ts

```diff
@@ -13,6 +13,8 @@ import {
 
 interface RuleOptions {
   maxSize?: number;
+  inlineMaxSize?: number;
+  externalMaxSize?: number;
```

Comment:
Hi @G-Rath, sorry for the delay to answer these comments. 

In this PR I also updated the schema definition for this rule, so it's impossible to have a definition like above because the JSON schema validation will throw an error. I understand your point, but even the docs of Jest refer to this type of snapshot as ["external"](https://jestjs.io/docs/en/snapshot-testing#inline-snapshots):

> "Inline snapshots behave identically to external snapshots (.snap files), except the snapshot values are written automatically back into the source code."

In this way, I think it is more explicit than it is being configured. However, if you still think the refactor is necessary I can do it.

Reply:
I did see the schema update, but that's also why I think it's worth going down this route: it's overall less complex, and still makes logical sense.

Here's my logic:
  * Dev has rule configured with `maxSize` set to `x`
  * Dev decides that they want inline snapshots to have max size of `y`

At this point, currently the flow is like this:
  * Dev adds `inlineMaxSize: y`
  * ESLint errors complaining about invalid configuration
  * Dev renames `maxSize` to `externalMaxSize`

By just reusing `maxSize` it would become:
  * Dev adds `inlineMaxSize: y`

-------

Thinking ahead a few moves, I think you're right down the line, as this feature actually raises another question: what if I only want to limit the size of *one* of the snapshot types?

Currently, if a size is not provided, we fallback to using `maxSize`, which has a default value of 50, meaning enabling this result requires you setting a size on both snapshot types, where as someone might want to only limit the size of inline snapshots for example.

*However* I think this is a distinctive enough use-case that we can ignore it for this PR, and implement it later if requested - generally I've found it's better to start with tighter rules and loosen over time by adding options as they're requested, rather than trying to prematurely guess what people want :)

Reply:
Okay, I'll refactor and remove `externalMaxSize`.